_posts/2021-02-24-cognito-demystified.markdown

```diff
@@ -0,0 +1,93 @@
+
+For each identity type, there is an assigned IAM role. This role has a policy attached to it which dictates which AWS services that role can access. When Amazon Cognito receives a request, the service will determine the identity type, determine the role assigned to that identity type, and use the policy attached to that role to respond. By modifying a policy or assigning a different role to an identity type, you can control which AWS services an identity type can access.
+
+This is how you are going to protect your money making content. When a user logs in to your app, authenticating via a User Pool, or directly via an Identity Pool, or as a guest, Amazon Cognito generates temporary AWS credentials for the user. These temporary credentials are associated with a specific IAM role associated the with the related authenticated or unauthenticated context. The associated IAM role lets you define a set of permissions to access your AWS resources. 
+
+By default, the Amazon Cognito Console creates IAM roles that provide access to Amazon Mobile Analytics and to Amazon Cognito Sync. Alternatively, you can choose to use existing IAM roles. For example, you may desire to extend these roles to protect API level content in AppSync (precious pump-and-dump predicitions), or static files in S3 (inspirational images of Elon Musk).
+
+The permissions in access policies attached to a role are effective across all users that assume that role. If you want to partition your users' access, you can do so via policy variables. Be careful when including your users' identity IDs in your access policies, particularly for unauthenticated identities as these may change if the user chooses to login. But this probably isn't the blog post to dig into that detail. Examples of and guidance for access policies can be found [here](https://docs.aws.amazon.com/cognito/latest/developerguide/iam-roles.html).
+
+
+### Stop Reading, Start Doing 😤
+
+Maybe not as concise as I thought! I haven't even spoken about User Pool Groups yet...nevertheless, hopefully cognito seems a bit less scary - there's plenty of information here to get you started. The best way of learning is doing, so if Cognito sounds like it could work for you, do it. Whats the worst that could happen? 😉
+<br /><br />
+
+
+
+> *Nate takes pride in delivering outcomes for his clients using modern software development tools and processes.Through his persistent effort to learn more about how technology fits into the world around us, Nate has developed a passion for showing empathy to better understand and solve business problems. But don’t let any of this fool you, it should not be a surprise when Nate engages you in a healthy debate as to why Jeff is, and will always be the best member of The Wiggles, when Nate routinely overestimates how many donuts he can fit in his mouth, or when Nate inevitably tells you a dirty joke.*
```

Comment:
Perhaps add a call to action (ie. a link to our get in touch page etc )

Reply:
see added call to action